docs/src/modules/components/AppNavDrawerItem.js

```diff
@@ -330,7 +330,7 @@ export default function AppNavDrawerItem(props) {
         {planned && <Chip label="Planned" sx={sxChip('grey')} />}
       </Item>
       {expandable ? (
-        <Collapse in={open} timeout={250} unmountOnExit>
+        <Collapse in={open} timeout="auto" unmountOnExit>
```

Comment:
Overall, this looks great! I think I'd just make them a bit faster ⎯ turning it back to `auto` makes them feel like the previous speed?!

Reply:
> I think I'd just make them a bit faster

You can increase 50 to something else. The default value is 36.

Reply:
Moved it to 150 ⎯ 50 is so fast that it felt like no animation at all. 😅 I'm feeling good about it now!

Reply:
Ah no, I meant `timeout="auto"` here but something higher than 50 in here: https://github.com/mui/material-ui/pull/38393/files#diff-e8ee699cc2a25b01080fce38f9be8e7646933d3134ebbf108c1ecc98d3f77a71R307.